app/views/organized/bikes/_search.html.haml

```diff
@@ -30,28 +31,24 @@
 
 %p.mt-4
   - if @search_query_present || params[:search_stickers].present?
-    %strong
-      = number_with_delimiter(@bikes.total_count)
-      = "bike".pluralize(@bikes.total_count)
-    matching
+    = t(".matching_bikes_html", bikes: pluralize(number_with_delimiter(@bikes.total_count), t(".bike")))
   - else
-    = number_with_delimiter(@bikes.total_count)
-    = "bike".pluralize(@bikes.total_count)
-    registered with #{current_organization.short_name}
+    = t(".registered_bikes_html",
+    bikes: pluralize(number_with_delimiter(@bikes.total_count), t(".bike")),
```

Comment:
you can pass `number_with_delimeter` to pluralize!?

Reply:
yeah! Only discovered that because I couldn't find any other way to get localized pluralization and delimiters in the same helper invocation. Happy surprise:

```ruby
helper.pluralize(helper.number_with_delimiter(1e10.to_i), 'year')
"10,000,000,000 years"
```